packages/kuflow-temporal-activity-kuflow/src/kuflow-activities-sync.ts

```diff
@@ -89,7 +86,7 @@ import {
   validateRetrieveTaskRequest,
   validateSaveProcessElementRequest,
   validateSaveTaskElementRequest,
-  validateSaveTaskJsonFormsValueData,
+  validateSaveTaskJsonFormsValueData
```

Comment:
Do you have a different eslint version ?

Reply:
Nop :face_in_clouds:

Reply:
If we want that rule, I add it to the eslint configuration.
`  'comma-dangle': ["error", "always-multiline"]`